src/Mvc/Mvc.Razor.RuntimeCompilation/src/build/netcoreapp3.0/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.targets

```diff
@@ -2,5 +2,8 @@
   <PropertyGroup>
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <PreserveCompilationReferences>true</PreserveCompilationReferences>
+
+    <!-- Generate hosting attributes during build time compilation to support runtime compilation -->
+    <GenerateRazorHostingAssemblyInfo Condition="'$(GenerateRazorHostingAssemblyInfo)'==''">true</GenerateRazorHostingAssemblyInfo>
```

Comment:
Hmm? Is this required for the C# lang stuff or just extra? Just curious

Reply:
Nope, it's unrelated. We removed it from Razor.SDK as part of https://github.com/aspnet/AspNetCore-Tooling/pull/389. I was waiting for the Razor build to be updated in this repo